### PR TITLE
Newsletter 3: Final draft

### PR DESCRIPTION
**Pull request by ozkriff**
[**Rendered**](https://github.com/ozkriff/rust-gamedev.github.io/blob/newsletter_003_draft/_drafts/newsletter-003.md)

The release is planned on ~~4, 5, 6 :(~~, 7 October 17:00 UTC

------

TODO:

- [x] Game Updates
  - [x] Sulis
  - [x] Veloren
  - [x] PF Sandbox
  - [x] Antorum
  - [x] Ferris Fencing
  - [x] Tennis Academy
  - [x] piano-rs
  - [x] Dissolve
  - [x] Garden
  - [x] Godot and Rust
    - [x] Rendering a 2D Game in 3D
    - [x] ardawanizadi's Month
    - [x] Pong Clone in Godot Using gdnative
  - [x] Ludum Dare 45
    - [x] Legally Dead
    - [x] Working Title
    - [x] Mindmaze
  - [x] Amethyst Games
    - [x] Arrakis
    - [x] webshinra's hex game
    - [x] Grumpy Visitors
    - [x] takeryo_eeic's hex game
    - [x] Space Shooter
  - [x] Other Game News
    - [x] Eve
    - [x] roboinstructus
- [x] Library & Tooling updates
  - [x] Dev Time Optimization
  - [x] RUSTFLAGS Tip
  - [x] RLSL
  - [x] gfx-rs
  - [x] Vulkano: Why Command Buffers Are So Complicated?
  - [x] splines-3.0.0
  - [x] Mun
  - [x] ultraviolet
  - [x] This month in rustsim 8
  - [x] cyclone-physics-rs
  - [x] Rust Roguelike Tutorial
  - [x] Nannou
  - [x] WIP Rendering Engine by Cynic64
  - [x] Amethyst
    - [x] specs
    - [x] amethyst_physics
    - [x] tutorial videos
- [x] Popular Workgroup Issues in Github
- [x] Requests for Contribution
- [x] Bonus

### CODE REVIEW

File: _drafts/newsletter-003.md [hunk 1]
```diff
@@ -96,26 +102,557 @@ _Discussions:
...
+[#38](https://veloren.net/devblog-38),
+[#39](https://veloren.net/devblog-39).
+
+[veloren]: https://veloren.net
+
+### PF Sandbox
+
+by [@rukai]
+
+- <https://twitter.com/thisIsRukai/status/1181208687081222154> -- textures!
```
Comment (kvark) [inline _drafts/newsletter-003.md]: I would prefer to have semi-equal space for games, and having to go look these links is a bit unfortunate.
Do you think we could embed a few images/gifs and maybe some words right here in the newsletter?

Comment (ozkriff) [reply _drafts/newsletter-003.md]: Yeah, this section is just a draft atm - see the TODO list in the PR's description.

File: _drafts/newsletter-003.md [hunk 2]
```diff
@@ -297,57 +297,57 @@ follow [@logicsoup] on Twitter.
...
-### [Godot][godot] and Rust
-
-- Rendering a 2D game in 3D
-
-  **TODO**: img
-
-  <https://medium.com/@recallsingularity/rendering-a-2d-game-in-3d-bd24ddbee6eb>
-
-  <https://twitter.com/RecallSingular1/status/1184059000242855936>
+### [EVE Online] Backend Optimization
```
Comment (AlexEne) [inline _drafts/newsletter-003.md]: This doesn't seem to come from the Eve Online team. Are we sure it's used in production?

Comment (ozkriff) [reply _drafts/newsletter-003.md]: Renamed

**Comment by ozkriff** [comment]
Finally, all content TODOs are gone. r?, please.

If everything is fine, I'll merge this today around 17:00 UTC.

**Comment by ozkriff** [comment]
@erlend-sh Have I forgot to mention any Amethyst's October news?

File: _drafts/newsletter-003.md [hunk 3]
```diff
@@ -96,26 +96,709 @@ _Discussions:
...
+
+[@heyrutvik]: https://twitter.com/heyrutvik
+[cyclone-physics-rs]: https://github.com/heyrutvik/cyclone-physics-rs
+[cyclone-video-demo]: https://twitter.com/heyrutvik/status/1180072669250834432
+[cyclone-physics-book]: https://www.crcpress.com/Game-Physics-Engine-Development-How-to-Build-a-Robust-Commercial-Grade/Millington/p/book/9780123819765
+
+### [Cynic64's WIP Rendering Engine][cynic64-ann]
+
+[@cynic64] shared a [demo video][cynic64-video] and
+a [report about their WIP rendeing engine][cynic64-ann].
```
Comment (Wodann) [inline _drafts/newsletter-003.md]:
```suggestion
a [report about their WIP rendering engine][cynic64-ann].
```

File: _drafts/newsletter-003.md [hunk 4]
```diff
@@ -129,17 +812,58 @@ or [join the next meeting][join].
...
 ------
 
+<!-- TODO: check that the first and the last image/gif looks nice:
+they may become a preview on different sites -->
+
 That's all news for today, thanks for reading!
 
 Want something mentioned in the next newsletter?
 [Send us a pull request][pr].
+Fell free to send PRs about your own projects!
```
Comment (Wodann) [inline _drafts/newsletter-003.md]:
```suggestion
Feel free to send PRs about your own projects!
```

**Comment by Wodann** [comment]
Well done. You managed to gather - and make insightful - an incredible amount of news again!

**Comment by kvark** [comment]
> The release is planned on 4, 5, 6 :(, 7 October 17:00 UTC

Come back to the future!

**Pull request merged**